### ISSUE TITLE
Exporting the HAP (or not)

### ISSUE THREAD

**Issue by lurid-bogey** [marcsowen/homebridge-homematicip#24]
TL;DR: Exporting the HAP to HomeKit is confusing. Don’t do it.

_Long version:_
Right now the Homematic Access Point (HAP) is exported to HomeKit as a dimmable light.

At first I was totally confused and couldn’t understand the reason behind this decision. Why would a computer be rendered as a light? What happens if I turn off this light? Do I turn off the HAP itself?

Later I found out that actually I can control the brightness of the LED and then the decision suddenly made sense. I spent quite a lot of time playing with it.

Now that the novelty has worn off, I feel that seeing the HAP in the Home app is definitely confusing.

The Home app reports “A light in the living room is on”. Then I start thinking “Why is a light still burning? I distinctly remember turning everything off!” After a moment of confusion I realize that actually I’m seeing the HAP, which is always on and cannot be turned off.

Anyway, my suggestion is either to not export the HAP to HomeKit at all, or at least to make it configurable.

**Comment by marcsowen** [comment]
It's been removed. You might need to delete the cache for the device to completely disappear. (I still haven't figured out how to do it automatically.)

**Comment by lurid-bogey** [comment]
Thank you. All is good now.
